test/unit/middleware/auth/sign_in_with_email_middleware_test.dart

```diff
@@ -0,0 +1,70 @@
+import 'package:crowdleague/actions/auth/sign_in_with_email.dart';
```

Comment:
This is brilliant!  It's not just some awesome tests but it's also providing some sweet documentation!

Reply:
true!